src/cmds/scripts/pbs_postinstall.in

```diff
@@ -99,31 +99,6 @@ createpath() {
 	done
 }
 
-backupdir() {
-	if [ ! -d "$1" -o ! -d "$2" ]; then
-		echo "*** Failed to backup $1 to $2"
-		return 1
-	fi
-	dir=`basename $1`
-	# Try using the timestamp from the directory itself
-	stamp=`date -r "$1" +%Y%m%d%H%M%S`
-	# If it exists, try the current time
-	[ -d "${2}/${dir}.${stamp}" ] && stamp=`date +%Y%m%d%H%M%S`
-	# If that exists, start from 000000 and work up
-	if [ -d "${2}/${dir}.${stamp}" ]; then
-		let i=0
-		while [ $i -lt 10000 ]; do
-			stamp=`printf '%04d' $i`
-			[ ! -d "${2}/${dir}.${stamp}" ] && break
-			let i+=1
-		done
-	fi
-	echo "*** Backing up $1 to ${2}/${dir}.$stamp"
-	echo "*** ${2}/${dir}.$stamp may need to be manually removed"
-	cp -pr "$1" "${2}/${dir}.$stamp" 2>&1
-	return 0
-}
-
```

Comment:
This piece of code is removed as we are not performing the backup activity any longer in pbs_postinstall.